### ISSUE TITLE
[Accessibility]A11y_VSCode_VSCLicenseAgreement_Keyboard: All the Links are not accessible by keyboard using tab key.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [Microsoft/vscode#51829]
**Environment Details:**
VSCode Version : 1.24.1
OS Version : Win10
**Additional Details:**
NVDA 2018.2, MAS2.1.1

**Repro Steps:**
1.On VSC Setup Wizard, click on "Next" button.
2.Check using tab key able to access links or  present inside (edit  read only  multi line dialogue box )

**Actual:**
Links are not accessible by keyboard using tab key (present inside (edit  read only  multi line dialogue box)

**Expected:**
All links should be accessible by keyboard using tab key present inside (edit  read only  multi line dialogue box )

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx

**User Impact:**
The user will not be able to access the controls using tab key.

**MAS Reference:**
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={8492c4eb-c179-40ae-8777-cd044ed725a2}

Does this issue occur when all extensions are disabled?: Yes
**Attachment for Reference:**
![accessibility a11y_vscode_vsclicenseagreement_keyboard](https://user-images.githubusercontent.com/39204909/41394423-8409d95e-6fc7-11e8-8522-de7b878504db.png)

**Comment by joaomoreno** [comment]
@AccessibilityTestingTeam-TCS This needs to go into https://github.com/jrsoftware/issrc
